src/main/scala/no/ndla/articleapi/controller/ArticleControllerV2.scala

```diff
@@ -332,6 +334,18 @@ trait ArticleControllerV2 {
       }
     }
 
+    private val Pattern = """(urn:)?(article:)?(\d*)#?(\d*)""".r
```

Comment:
Eg tenkte meir at id her kunne være id#rev. urn:article strippes allerede bort i feks graphql. Men når eg tenker meg om så er det vel tatt høgde for det i regexpen.

Reply:
Riktig, denne håndterer begge caser